### PR TITLE
Fix bug handling boolean as string

### LINKED ISSUE #141: delete default set to bool False, but is handled as a string
**Describe the bug**

```
Traceback (most recent call last):
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 210, in reply_and_send
    reply = fun(message)
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 131, in push
    new_msg = self._move_files(message)
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 148, in _move_files
    error_message = self._move_file(pathname, message, rel_path)
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 170, in _move_file
    self._add_to_deleter(pathname)
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 174, in _add_to_deleter
    if self._attrs.get('compression') or self._is_delete_set():
  File "/software/miniconda/envs/pytroll/lib/python3.9/site-packages/trollmoves/server.py", line 178, in _is_delete_set
    return self._attrs.get('delete', 'False').lower() in ["1", "yes", "true", "on"]
AttributeError: 'bool' object has no attribute 'lower'
```
**To Reproduce**
You need to run the server

**Expected behavior**
To work even if delete is not set

**Environment Info:**
 - OS: Linux
 - Trollmoves Version: main/master

### PR DESCRIPTION
**Pull request by TAlonglong**
- [X] Closes #141  
 - [X] Tests added  
 - [ ] Fully documented 
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Comment by TAlonglong** [comment]
Hm, this fixes only if the config is not yaml. Not sure what happens if the config is yaml.

File: trollmoves/tests/test_server.py [hunk 1]
```diff
@@ -282,3 +282,21 @@ def test_requestmanager_run_MessageError_exception(patch_validate_file_pattern,
...
         req_man._run_loop()
     assert "Failed to create message from payload: fake_payload with address address" in caplog.text
+
+
+@patch("trollmoves.server.RequestManager._validate_file_pattern")
+def test_requestmanager_is_delete_set(patch_validate_file_pattern):
+    """Test delete default config"""
+    from trollmoves.server import RequestManager
+    port = 9876
+    req_man = RequestManager(port, attrs={'delete': False})
```
Comment (pnuu) [inline trollmoves/tests/test_server.py:292]: Shouldn't the default test be without `'delete'` being set?
```suggestion
    req_man = RequestManager(port, attrs={})
```

**Comment by TAlonglong** [comment]
Thanks @pnuu. Applied your suggestions

**Review by pnuu — APPROVED** [review]
LGTM

**Pull request merged**